apps/chat/src/components/Folder/Folder.tsx

```diff
@@ -779,14 +788,20 @@ const Folder = <T extends ConversationInfo | PromptInfo | DialFile>({
     }
   }, [currentFolder.id, dispatch, featureType, searchTerm]);
 
+  const isInCurrentPublication =
```

Comment:
It might be better to name this variable IsPartOfSelectedPublication or something like that